examples/keygen.php

```diff
@@ -1,20 +1,24 @@
 <?php
 
+use phpseclib3\Crypt\RSA;
+use phpseclib3\Crypt\RSA\Formats\Keys\PKCS1;
+
 @include_once dirname(__FILE__).'/../vendor/autoload.php';
 require_once dirname(__FILE__).'/../lib/openpgp.php';
 require_once dirname(__FILE__).'/../lib/openpgp_crypt_rsa.php';
 
-$rsa = new \phpseclib\Crypt\RSA();
-$k = $rsa->createKey(512);
-$rsa->loadKey($k['privatekey']);
+$privateKey = RSA::createKey(512);
+$publickey = $privateKey->getPublicKey();
+
+$privateKeyComponents = PKCS1::load($privateKey->toString('PKCS1'));
 
 $nkey = new OpenPGP_SecretKeyPacket(array(
-   'n' => $rsa->modulus->toBytes(),
-   'e' => $rsa->publicExponent->toBytes(),
-   'd' => $rsa->exponent->toBytes(),
-   'p' => $rsa->primes[2]->toBytes(),
-   'q' => $rsa->primes[1]->toBytes(),
-   'u' => $rsa->coefficients[2]->toBytes()
+   'n' => $privateKeyComponents["modulus"]->toBytes(),
+   'e' => $privateKeyComponents["publicExponent"]->toBytes(),
+   'd' => $privateKeyComponents["privateExponent"]->toBytes(),
+   'p' => $privateKeyComponents["primes"][2]->toBytes(),
+   'q' => $privateKeyComponents["primes"][1]->toBytes(),
+   'u' => $privateKeyComponents["coefficients"][2]->toBytes()
```

Comment:
the properties are no more public  , so the best way is to convert the private key in its "raw array" form